Change so to source_display_name

emma-wilson opened the pull request:
Fixes issue https://github.com/camaradesuk/solesR/issues/34

emma-wilson committed:
Change so to source_display_name

seancsmith reviewed (changes requested):
This part of the function still doesn't seem to work. I think the author column may have changed as well

correct_doi <- match %>%
        tidyr::unnest(cols=author.y) %>%
        filter(author_position == "first") %>%
        rename(author_orig = author.x,
               new_doi=doi.y)...

emma-wilson commented:
Thanks for spotting, I'll try to fix this afternoon

emma-wilson committed:
Fix authorship columns

emma-wilson commented:
@seancsmith I think I've updated authorships -- want to have another go at testing?

emma-wilson committed:
fix error in column name

emma-wilson committed:
fix column names and use year instead of page match

emma-wilson commented:
Okay, finally ready for re-review. I've tested and everything is working on my end. Changes made:

- Column names are now properly tested and fixed
- Slide_head added to remove duplicate DOIs
- The function needs a publication year match rather than a page match (I remember we added page match for a conference abstract issue but in my tested i found it way too restrictive and year works much better -- I believe switching to year match was suggested by Maria before too).
